Documentation/AndroidSimulator.md

```diff
@@ -12,3 +12,18 @@ The HAXM installer can fail for a number of reasons. Many solutions are discusse
 - Disabling Hyper-V in Windows
 - Disabling memory integrity in Windows
 - Disabling virtualization security with the Device Guard and Credential Guard hardware readiness tool
+
+### OpenGL ES 3 and AR settings
+
+Open GL ES3 is required on Android Emulator. Otherwise one will run into the following errors when deploying it onto the Android Emulator:
```

Comment:
I don't need to do this on my machine, but I also have some weird issues with the Android Emulator. Changing to Open GL ES3 does nothing for me though.